mmv1/third_party/terraform/services/composer/resource_composer_environment.go.erb

```diff
@@ -27,7 +27,7 @@ import (
 const (
 	composerEnvironmentEnvVariablesRegexp          = "[a-zA-Z_][a-zA-Z0-9_]*."
 	composerEnvironmentReservedAirflowEnvVarRegexp = "AIRFLOW__[A-Z0-9_]+__[A-Z0-9_]+"
-	composerEnvironmentVersionRegexp               = `composer-(([0-9]+)(\.[0-9]+\.[0-9]+(-preview\.[0-9]+)?)?|latest)-airflow-(([0-9]+)((\.[0-9]+)(\.[0-9]+)?)?)`
+	composerEnvironmentVersionRegexp               = `composer-(([0-9]+)(\.[0-9]+\.[0-9]+(-preview\.[0-9]+)?)?|latest)-airflow-(([0-9]+)((\.[0-9]+)(\.[0-9]+)?)?(-build\.[0-9]+)?)`
```

Comment:
Could you give an example of what's being supported by this change?

Reply:
In Composer 3 users will be able to optionally specify the Airflow build number (e.g composer-3-airflow-2.6.3-build.4). Build number is always returned in environment details by the API. I added this to make sure full version is parsed and versions are compared properly in case of updates.